airmozilla/main/static/main/css/event.css

```diff
@@ -95,3 +95,62 @@ p.pending {
 .playbackrate .options a.current {
     color: white;
 }
+.tabs{
+    position: relative;
+    min-height: 200px;
+    clear: both;
+    margin: 25px 0;
+}
+.tabs {
+  position: relative;
+  min-height: 200px; /* This part sucks */
+  clear: both;
+  margin: 25px 0;
+}
+.tab {
+      float: left;
+}
+.tab label {
+      background: #20272a;
+      padding: 8px;
+      border: none;
+      margin-left: 0px;
+      position: relative;
+      left: 1px;
+}
+.tab [type=radio] {
+      display: none;
+}
+.content {
+  position: absolute;
+  top: 28px;
+  left: 0;
+  background-color: #2d373b;
+  right: 0;
+  bottom: 0;
+  padding: 20px;
+  border-bottom-right-radius: 5px;
+  border-bottom-left-radius: 5px;
```

Comment:
Lets remove the bottom rounded corner. I think it would look strange if the tab thing has rounded corners and the sidebar thing below doesn't.